src/controllers/imposterController.js

```diff
@@ -31,7 +31,7 @@ function create (imposters) {
             query = url.parse(request.url, true).query,
             options = { replayable: queryBoolean(query, 'replayable'), removeProxies: queryBoolean(query, 'removeProxies') },
             imposter = imposters[request.params.id].toJSON(options);
-
+        recordImposter(imposter);
```

Comment:
Why record on a GET? This operation isn't altering any state.

Reply:
This is because when we use replayable and removeproxies, the saved structure will get filtered and saved to the file.  We messed around with this a bit and thought this was the best option.

Reply:
got it, thanks